tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 envlist =
-       {py27,py34}-django{1.8,1.9,1.10}-drf{3.3,3.4,3.5}
+       {py27,py34,py35,py36}-django{1.8,1.9,1.10}-drf{3.3,3.4,3.5}
```

Comment:
Good catch, I've updated the PR!

The travis tests has run with DRF 3.6 anyways since it is included in the Travis test list.